st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -144,7 +144,10 @@ def try_run(self, action_parameters):
         inputs = self.runner_parameters.get('context', dict())
         inputs.update(action_parameters)
 
-        endpoint = 'http://%s:%s/v1/actionexecutions' % (cfg.CONF.api.host, cfg.CONF.api.port)
+        # If host value does not start with http, then default to http://.
+        protocol = 'http' if not cfg.CONF.api.use_ssl else 'https'
```

Comment:
Another problem (lack of consistency) with calling this option `use_ssl` is that `auth`service / section also exposes an option with the same name, but it serves a different purpose.

If this option is enabled, auth API service listens on https when running in a standalone mode. So this lack of consistency could cause some confusion.